cranelift-codegen/meta/src/isa/x86/recipes.rs

```diff
@@ -788,6 +790,48 @@ pub fn define<'shared>(
         );
     }
 
+    // XX /r ib with 8-bit unsigned immediate (e.g. for pshuf)
+    {
+        let format = formats.get(f_extract_lane);
+        recipes.add_template_recipe(
+            EncodingRecipeBuilder::new("r_ib_unsigned", f_extract_lane, 2)
```

Comment:
I'm not sure about the naming of these templates: `r == register`? `ib == immediate byte`?

Reply:
Yes, sounds consistent with what's in there.